### ISSUE TITLE
Show ROI or APR of each pool on the home page

### ISSUE THREAD

**Issue by penandlim** [yieldfarming/yield-farming#2]

**Comment by Deepcryptodive** [comment]
I agree, this might be really handy and prevents people generating unnecessary bandwith opening each link to see their returns.
Something simple, similar to what Curve.fi has, would be great.
![Screenshot from 2020-07-27 15-50-48](https://user-images.githubusercontent.com/61325205/88549746-fa97ba80-d020-11ea-915b-60332e3a41dc.png)

**Comment by DarkGhost7** [comment]
I think this should be added if you have time.
